examples/di_hw_timed.py

```diff
@@ -0,0 +1,42 @@
+﻿"""Example for reading digital signal."""
+
+import pprint
+
+import nidaqmx
+from nidaqmx.constants import LineGrouping
+
+pp = pprint.PrettyPrinter(indent=4)
+
+
+with nidaqmx.Task() as task:
+    task.di_channels.add_di_chan("Dev1/port0/line0", line_grouping=LineGrouping.CHAN_PER_LINE)
+    task.timing.cfg_samp_clk_timing(1000)
+
+    print("1 Channel 1 Sample Read: ")
+    data = task.read()
+    pp.pprint(data)
+    task.wait_until_done()
+    task.stop()
+
+    print("1 Channel N Samples Read: ")
+    data = task.read(number_of_samples_per_channel=8)
+    pp.pprint(data)
+    task.wait_until_done()
+    task.stop()
```

Comment:
delete, one thing per example

Reply:
I think it would be good to update the existing examples to demonstrate one thing per example. :)